entries/w/wisc.edu.json

```
@@ -0,0 +1,17 @@
+{
+    "University of Wisconsin Madison": {
+        "domain": "wisc.edu",
+        "url": "https://www.wisc.edu/",
+        "tfa": [
+            "custom-software",
+            "custom-hardware"
+        ],
```

The custom hardware and software offered needs to be specified:
```suggestion
        ],
        "custom-software": [
          "Duo"
        ],
        "custom-hardware": [
          "Duo TOTP key"
        ]
```